packages/devextreme/js/__internal/grids/tree_list/data_source_adapter/m_data_source_adapter.ts

```diff
@@ -338,22 +345,29 @@ let DataSourceAdapterTreeList = DataSourceAdapter.inherit((function () {
     },
 
     _loadParentsOrChildren(data, options, needChildren) {
-      const that = this;
       let filter;
       let needLocalFiltering;
-      const { keys, keyMap } = that._generateInfoToLoad(data, needChildren);
+      const { keys, keyMap } = this._generateInfoToLoad(data, needChildren);
       // @ts-expect-error
       const d = new Deferred();
       const isRemoteFiltering = options.remoteOperations.filtering;
-      const maxFilterLengthInRequest = that.option('maxFilterLengthInRequest');
+      const maxFilterLengthInRequest = this.option('maxFilterLengthInRequest');
+      const sort = options.storeLoadOptions?.sort ?? options.loadOptions?.sort;
       let loadOptions = isRemoteFiltering ? options.storeLoadOptions : options.loadOptions;
```

Comment:
I think you can access the sort options directly from `loadOptions`.
```
applySorting(
    this._cachedStoreData.concat(loadedData),
    loadOptions.sort,
)
```
This way, additional variable to hold the option is not required.

Reply:
I can get the sort option from options.loadOptions or options.storeLoadOptions depending on the remoteOperations.sorting option and not from remoteOperations.filtering

So, I won't be able to access it from loadOptions variable.

Reply:
> const isRemoteFiltering = options.remoteOperations.filtering;

Right, I missed this part. Thanks.